crates/sui/tests/snapshots/cli_tests__body_fn.snap

```diff
@@ -0,0 +1,21 @@
+---
+source: crates/sui/tests/cli_tests.rs
+expression: "format!(\"Non-clever-abort\\n---\\n{}\\n---\\nLine-only-abort\\n---\\n{}\\n---\\nClever-error-utf8\\n---\\n{}\\n---\\nClever-error-non-utf8\\n---\\n{}\\n---\\n\",\n    non_clever_abort, line_only_abort, clever_error_utf8,\n    clever_error_non_utf8)"
+---
+Non-clever-abort
+---
+Error executing transaction: 1st command aborted within function '0xc9d9682e45a4291cf2c7d86486f604fe19451bb367cc9fa4f2149eb5f1efac8a::clever_errors::aborter' at instruction 1 with code 0
```

Comment:
Do we have an address map or anything at this point?

Reply:
I don't believe we do. But I'll look around to see -- I could look in the CLI address aliases and possibly look about using them if an address matches.

Reply:
Pardon my noob question, but isn't this a package address? That would not exist in the CLI address aliases, because it only host wallet addresses' aliases?

Reply:
😢 Nope, you're totally right. Nothing we can really do about it then (at least right now).